src/routes/+layout.svelte

```diff
@@ -1,77 +1,73 @@
 <script lang="ts">
-	import "@brainandbones/skeleton/themes/theme-skeleton.css";
-	import "@brainandbones/skeleton/styles/all.css";
-	import "../app.postcss";
-	import { page } from "$app/stores";
-	import { selectedLanguagesStore } from "../common/sideBarContents.svelte";
-	import {
-		AppShell,
-		AppBar,
-		GradientHeading,
-		Drawer,
-	} from "@brainandbones/skeleton";
-	import { writable, type Writable } from "svelte/store";
-	import SideBarContents from "../common/sideBarContents.svelte";
+  import "@brainandbones/skeleton/themes/theme-skeleton.css";
+  import "@brainandbones/skeleton/styles/all.css";
+  import "../app.postcss";
+  import { page } from "$app/stores";
+  import { selectedLanguagesStore } from "$lib/common/sideBarContents.svelte";
```

Comment:
This line is giving an error.

![Screenshot from 2022-11-25 21-21-06](https://user-images.githubusercontent.com/10581984/204020003-6dcbc9aa-2a14-440a-b318-8bfec15d9d1e.png)

Reply:
Try running `npx svelte-kit sync` (or `yarnpkg` etc.), that should fix it.

Reply:
Thank you!